src/api/registerForEvents.js

```diff
@@ -11,7 +11,6 @@ export default (auth: Auth) =>
         'update_message',
         'subscription',
         'reaction',
-        'presence',
```

Comment:
Are you sure? That's not the behavior I'm seeing. When I include 'presence', the result includes a 'presences' field (which is enormous). When I exclude 'presence', the result does not include that field